packages/api/src/routers/block/getByPartialHash.ts

```diff
@@ -0,0 +1,20 @@
+import { prisma } from "@blobscan/db";
+
+export async function getByPartialHash(hash: string): Promise<string | null> {
```

Comment:
Let's rename this to something more descriptive.

```suggestion
export async function getFullBlockHash(partialHash: string): Promise<string | null> {
```